spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -326,32 +326,10 @@
 
         it "allows user to join" do
           patch :accept_invitation, params: { id: invitation.key, group: { id: group.id } }
-          expect(group_assignment.group_assignment_repos.count).to eql(1)
           expect(student.repo_accesses.count).to eql(1)
         end
       end
 
-      context "github repository with the same name already exists" do
```

Comment:
These tests can be removed because `#accept_invitation` no longer creates a GitHub repository after the `group_import_resiliency` changes.